chain/network/src/config.rs

```diff
@@ -174,18 +198,21 @@ impl NetworkConfig {
             blacklist: cfg
                 .blacklist
                 .iter()
-                .map(|e| e.parse().expect("failed to parse blacklist"))
-                .collect(),
+                .map(|e| e.parse())
+                .collect::<Result<_, _>>()
+                .context("failed to parse blacklist")?,
```

Comment:
This feels like a syntactic rule which can be part of Config parsing. Out of scope for the PR.

Reply:
yes, there is a bunch of places like that